release-notes/elastic-security/known-issues.md

```diff
@@ -8,14 +8,35 @@ Known issues are significant defects or limitations that may impact your impleme
 % Use the following template to add entries to this page.
 
 % :::{dropdown} Title of known issue
-% Applies to: Applicable versions for the known issue 
+% Applies to: Applicable versions for the known issue
 % Description of the known issue.
 % For more information, check [Issue #](Issue link).
 % **Impact**<br> Impact of the known issue.
 % **Workaround**<br> Steps for a workaround until the known issue is fixed.
 
 % :::
 
+:::{dropdown} Elastic Defend's Network driver may lead to bug checks
+
+**Applies to: {{agent}} 8.17.8, 8.18.3, and 9.0.3**
```

Comment:
Typically we would only document known issues for 9.x here and 8.x known issues in the [8.18](https://www.elastic.co/guide/en/fleet/8.18/release-notes-8.18.2.html) and [8.17](https://www.elastic.co/guide/en/fleet/8.17/release-notes-8.17.7.html) docs. However, since we're currently blocked from updating those pages, I think it's ok to add the 8.x versions here for now.